pkg/virtctl/vm/vm.go

```diff
@@ -138,6 +143,13 @@ func (o *Command) Run(cmd *cobra.Command, args []string) error {
 			return fmt.Errorf("Error stopping VirtualMachine %v", err)
 		}
 	case COMMAND_RESTART:
+		if forceRestart {
```

Comment:
Did you intentionally decide to not expose `--grace-period`?

Reply:
First was not going to include it in `virtctl`, but then we expose it in api anyway, so included it now